### DISCUSSION TITLE
Feature Request: make totem of undying an amulet/add compatibility with backhand slot

### DISCUSSION THREAD

**Discussion by huihiuhuai** [Roadhog360/Et-Futurum-Requiem#479] [Improvements]
### Initial Questions

- [X] I have searched this issue tracker and there is nothing similar already, and this is not on the [list of known issues](https://github.com/Roadhog360/Et-Futurum-Requiem/issues/146). Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I have verified that this issue occurs in a SUPPORTED environment, meaning I can reproduce this WITHOUT OptiFine, and without Bukkit+Forge server software like KCauldron, Thermos, Crucible, Mohist etc

### Mod Version

 2.6.0

### Describe the Issue

title

**Comment by Roadhog360** [comment]
Issues are for issues, not feature requests. Moving to the correct place

**Comment by Roadhog360** [comment]
This is actually a really nice idea though, I was waiting for offhand but I didn't think of doing it this way.
